CHANGELOG.md

```diff
@@ -155,7 +155,7 @@ sequential patch number (starting from `01`).
     were written by default.
   - FMS2_io does not do this. Users can specify real long_names and units by calling register_variable_attribute.
 ### Removed
-- FMS_io was almost completely removed from FMScoupler and replaced with fms2_io. 
+- FMS_io was almost completely removed from FMScoupler and replaced with fms2_io.
```

Comment:
What is still remaining?

Reply:
We have ifdef'd calls to `fms_io_exit` at the end of all the coupler versions: https://github.com/NOAA-GFDL/FMScoupler/blob/0b2c91402236e49d586dd5151c98d329e97f9e57/full/full_coupler_mod.F90#L1315-L1317

I don't think they can be removed yet due to the possibility of a component using the old io.

Reply:
Can we put that in as a sub-bullet to describe what is still in place?

Reply:
Yeah definitely, added with 408f90d